org.kde.akregator.json

```
@@ -0,0 +1,495 @@
+{
+    "id": "org.kde.akregator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "akregator",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--own-name=org.freedesktop.Akonadi",
+        "--own-name=org.freedesktop.Akonadi.*",
+        "--own-name=org.kde.kaddressbook",
+        "--own-name=org.kde.accountwizard",
+        "--own-name=org.kde.akregator",
+        "--own-name=org.kde.pimsettingsexporter",
+        "--own-name=org.kde.pim.*",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--talk-name=org.kde.kded5",
+        "--talk-name=org.kde.kglobalaccel",
+        "--talk-name=org.kde.JobViewServer",
+        "--talk-name=org.kde.klauncher5",
+        "--talk-name=org.kde.knotify",
+        "--talk-name=org.kde.kwalletd5",
+        "--talk-name=org.kde.kcookiejar5",
+        "--system-talk-name=org.freedesktop.NetworkManager"
+    ],
+
+    "modules": [
```

```suggestion
    "cleanup-commands": [
        "# Cleanup after QtWebEngine",
        "rm -rf /app/include",
        "rm -rf /app/lib/{cmake,mkspecs,pkgconfig} /app/lib/*/*.la"
    ],
    "modules": [
```

Cleanup after webengine